server/propertymanager/views.py

```diff
@@ -33,8 +33,9 @@ class DefaultSchema(schema_cls):
         inservice_date = fields.Date()
         expected_life = fields.Number()
 
-    schemas = SchemaContainer(DefaultSchema)
-
+    schemas = SchemaContainer(DefaultSchema, retrieve=DefaultSchema)
+    def retrieve(self, request, *args, **kwargs):
+        return super().retrieve(request, args, kwargs)
```

Comment:
this is not good, the retrieve method should be smart enough that this is not needed. why is this needed? how can it be eliminated?